auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -202,6 +247,9 @@ public FileSystemSecurityRealm(final Path root, final NameRewriter nameRewriter,
         this.hashEncoding = hashEncoding != null ? hashEncoding : Encoding.BASE64;
         this.providers = providers != null ? providers : INSTALLED_PROVIDERS;
         this.secretKey = secretKey;
+        this.privateKey = privateKey;
+        this.publicKey = publicKey;
+        this.mainIndex = new File(root.toString(),"main_index");
```

Comment:
just a minor,  would be good to add a javadoc for these. Especially mainIndex is not immediately clear